packages/yoga/src/Avatar/web/AvatarCircle.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { UserFilled } from '@gympass/yoga-icons';
+import Avatar from './Avatar';
+
+const AvatarCircle = props => (
+  <Avatar {...props} borderRadius="circle" placeholder={UserFilled} />
```

Comment:
This way we can't use anything but the `UserFilled` icon with the `Avatar.Circle` component. Do we even need this "default" behavior?

Even the scenario on our spec also uses the same icon and that spec only passes because it is a snapshot matching.

Reply:
You're absolutely right! Thanks

Reply:
Our design guidelines show us that when we are using `Avatar.Circle` the placeholder must be the `UserFilled`.With this in mind, we are defining the `placeholder` and `borderRadius` to reflect those decisions. 
Any other Avatar you may need, you can achieve it by just using Avatar and them providing some props.